tools/third_party/py/bench/localpath.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
https://github.com/pytest-dev/pytest/ seems alive still.

Reply:
Yeah, nothing in `tools/third_party` should be touched.